integration-cli/docker_cli_volume_test.go

```diff
@@ -212,7 +212,7 @@ func (s *DockerSuite) TestVolumeCliRm(c *check.C) {
 func (s *DockerSuite) TestVolumeCliNoArgs(c *check.C) {
 	out, _ := dockerCmd(c, "volume")
 	// no args should produce the cmd usage output
-	usage := "Usage:	docker volume [OPTIONS] [COMMAND]"
+	usage := "Usage:	docker volume COMMAND"
```

Comment:
Since there are options (even if just the default one of `-h`), I would think the more correct syntax would be to include `[OPTIONS]` otherwise its not clear if it should be `docker -h volume` or `docker volume -h`